test/integration/edit_int_test.go

```diff
@@ -43,14 +44,28 @@ scripts:
 
 	editorScriptDir := filepath.Join(ts.Dirs.Work, "editor")
 
+	locator := "which"
+	env := []string{}
 	var extension string
 	if runtime.GOOS == "windows" {
 		extension = ".exe"
+		locator = "where"
+		// The LOCALAPPDATA env var is used by the go binary to locate the cache dir
+		env = append(env, "LOCALAPPDATA="+ts.Dirs.TempDir)
 	}
+
+	// Locate the go binary as it will not be on PATH in the sandboxed environment
+	cmd := exec.Command(locator, "go")
+	output, err := cmd.Output()
+	suite.Require().NoError(err, "go binary not found in path: %s", string(output))
+	goBinary := string(output)
+	goBinary = strings.TrimSpace(string(goBinary))
```

Comment:
I can see other tests needing this, so perhaps we should move this into the testhelpers/e2e package? You might have to update the default session env var list though. It might get messy.